src/main/java/seedu/address/logic/parser/AddCommandParser.java

```diff
@@ -44,10 +46,10 @@ private static boolean arePrefixesPresent(ArgumentMultimap argumentMultimap, Pre
      */
     public AddCommand parse(String args) throws ParseException {
         ArgumentMultimap argMultimap =
-            ArgumentTokenizer.tokenize(args, PREFIX_NAME, PREFIX_PHONE, PREFIX_EMAIL, PREFIX_ADDRESS, PREFIX_TAG,
-                PREFIX_CARS);
+            ArgumentTokenizer.tokenize(args, PREFIX_NAME, PREFIX_PHONE, PREFIX_EMAIL, PREFIX_ADDRESS, PREFIX_DOB,
+                PREFIX_TAG, PREFIX_CARS);
```

Comment:
Side note, I notice we didn't really use Tag attribute from AB3.
Any ideas to use it or just leave it